worker/codalabworker/aws_batch.py

```
@@ -509,7 +528,7 @@ def update(self):
         if status == BatchStatus.Succeeded:
             run_status = 'Succeeded'
             finalize_message = {
-                'exitcode': container.get('exitCode'),
+                'exitcode': container.get('exitCode', 0),
```

Why default to 0? Seems like it could hide something going wrong?

---

This is inside a match on the job status as reported by batching being "succeeded". I believe it is the case that there isn't always an exit code set however I'm not sure of that and it's been too long for me to be comfortable touching this so I'm going to leave it as-is.